### ISSUE TITLE
Feature Request; Radio Playlist Overhaul

### ISSUE THREAD

**Issue by Korrivyn** [SkytAsul/JukeBox#4]
Howdy! I'd love to see a feature to allow staff with a specific permission node (e.g. music.command.admin.radio) to have control over the radio (next song, pause/play, choose a specific song, etc). Furthermore having the ability to set a playlist for the radio that doesn't have to include all existing songs would be great so some songs can be reserved for specific situations or for players to listen to on their own instead of on the global radio.

Essentially having more preset playlist options would be the crux of this update, which could be expanded upon to allow multiple stored playlists that a player can choose from, or a radio-controlling staff could choose from. If this were brought about this plugin could become infinitely more useful for many servers that rely on the atmosphere to capture an audience (RPG, MMO, RP, etc).

So, to summarize in point-form;
- Add Server-Owned playlists that can be chosen from instead of *just* music discs.
- Add permission to view all songs, just playlists, or maybe even specific songs/playlists.
- Rework the radio to run off a pre-determined playlist from the above system (set in config)
- Add permission to allow users to modify the radio's current playing as if it was their personal playlist.
- Ensure that a user selecting a singular song from *any* playlist will still return the radio to it's set playlist after.

I hope this all makes sense, and if not please don't hesitate to ask for clarification! Once I'm done some other projects I might fork and PR to help out if it ends up being problematic.
